vercel/examples
# Fix/missing paren plop

### Description

Fixes a missing parenthesis in the plop template that was causing new example deploy buttons to be misaligned. also fixes it for the on-demand-isr example

### Best Way to Test

- create a new example with the plop commands
- check the rendered readme to ensure it's not misaligned
- 
### Type of Change


### New Example Checklist
